designs/index.md

```diff
@@ -56,13 +53,18 @@ current Bazel functionality.
 
 ## Reasons for a design review policy
 
+Bazel has a design review policy:
+
 *   To maximize the likelihood that all feature requests get a baseline level of
     scrutiny.
 *   To maximize the likelihood that the right people will weigh in on designs
     before we've invested in an implementation that may not work.
 
```

Comment:
Did a little rearranging as promised with the intro section. PTAL at this proposed reordering and make any changes that you see fit. (The line wrapping is probably rather messed up, so apologies if this doesn't paste cleanly.

# Design Documents

If you're planning to add, change, or remove a user-facing feature, or make a
*significant architectural change* to Bazel, you **must** write a design document
and have it reviewed before you can submit the change.

Here are some examples of significant changes:

*   Addition or deletion of native build rules
*   Breaking-changes to native rules
*   Changes to a native build rule semantics that affect the behavior of more
    than a single rule
*   Changes to Bazel's rule definition API
*   Changes to the APIs that Bazel uses to connect to other systems
*   Changes to the Starlark language, semantics, or APIs
*   Changes that could have a pervasive effect on Bazel performance or memory
    usage (for better or for worse)
*   Changes to widely used internal APIs
*   Changes to flags and command-line interface.

## Reasons for design reviews

Having a design helps you to coordinate work with other Bazel developers and seek guidance from Bazel's core team. For example, if a proposal adds, removes, or modifies any function or object available in BUILD, WORKSPACE, or bzl files,  you'd add the [Starlark team](../maintainers-guide.html) as reviewers. Design documents are reviewed before submission because:

*   Bazel is a very complex system; seemingly innocuous local changes can have
    significant global consequences.
*   The team gets many feature requests from users; such requests need to be
    evaluated not only for technical feasibility but importance with regards to
    other feature requests.
*   Bazel features are frequently implemented by people outside the core team;
    such contributors have widely varying levels of Bazel expertise.
*   The Bazel team itself has varying levels of expertise; no single team member
    has a complete understanding of every corner of Bazel.
*   Changes to Bazel must account for backward compatibility and avoid breaking
    changes.

Bazel's design review policy helps to maximize the likelihood that:

*   all feature requests get a baseline level of scrutiny.
*   the right people will weigh in on designs before investing in an implementation that may not work.

To help you get started, take a look at the list of design documents in the [Bazel Proposals Repository](https://github.com/bazelbuild/proposals). Designs are works in progress so their implementation details can change over time and with feedback. The
published design documents capture the initial design, and _not_ the ongoing
changes as designs are implemented. Always go to the
[documentation]({{ site.docs_site_url }}) for descriptions of
current Bazel functionality.

Reply:
Done, with very minor edits.